app/views/content_items/service_manual_guide.html.erb

```diff
@@ -78,5 +78,53 @@
       <%= render 'govuk_component/govspeak', content: @content_item.body.html_safe %>
     </div>
 
+    <div class="change-history" data-module="toggle">
+      <div class="change-history__latest-change">
+        <dl>
+        <% if @content_item.content_owners.any? %>
+          <dt>Published by:</dt>
+          <% @content_item.content_owners.each do |content_owner| %>
+            <dd>
+              <%= link_to content_owner.title, content_owner.href %>
+            </dd>
+          <% end %>
+        <% end %>
+        <% if @content_item.latest_change.present? %>
+          <dt>Last update:</dt>
+          <dd>
+            <% time = @content_item.latest_change.public_timestamp %>
+            <time datetime="<%= time.iso8601 %>" class="timestamp"><%= time.strftime("%e %B %Y") %></time>
```

Comment:
:clap: for the use of the time element.

To get that strftime out of the view, this is the Rails-y way to do it: http://guides.rubyonrails.org/i18n.html#adding-date-time-formats